.circleci/config.yml

```
@@ -176,6 +176,7 @@ jobs:
           command: |
             ./gradlew --no-daemon --parallel integrationTest
       - capture_test_results
+      - capture_test_artifacts
```

Ah, this shouldn't work and the fact it does is actually a bug in the acceptance test job.

`capture_test_artifacts` is meant to capture `build/test-artifacts` (the artefacts specifically written by ATs) but has the path incorrectly set to `build/test-results`.  I think you want here:
```suggestion
      - capture_test_results
      - store_artefacts:
          path: build/test-results
```

and then we should fix the path in `capture_test_artifacts`.